### ISSUE TITLE
Generalise to Python (and Docker?)

### ISSUE THREAD

**Issue by wbryant** [nhs-r-community/statements-on-tools#9]
The arguments around R and R packages will be similar for Python and Docker, though their repos (e.g. PyPi and Dockerhub) are run somewhat differently to CRAN as I understand it.

Can this therefore be generalised, but with a few R/Python specific chunks mixed in there.  

Not to overcomplicate things but I just discovered RMarkdown parameters; I could just imagine having a document here where you can select Python or R at the beginning and the entire document adjusts accordingly ...

**Comment by ChrisBeeley** [comment]
I agree with you, I wouldn't want to over-generalise it though. I think it would be good to have something that goes:

* Here is guidance for open tooling and code sharing
* Here is guidance for R
* Here is guidance for Python
* Here is guidance for Docker

However, I wouldn't want the NHS-R community to produce guidelines for using Python or Docker, I'd like help from others to do this
